sultan/app/page.scss

```
@@ -0,0 +1,12 @@
+@import './styles/vars.scss';
```

```suggestion
@import './styles/vars';
```
Можно не указывать разрешение.

---

Не стоит отмечать выполненными замечания, которые не были выполнены. Отнеситесь, пожалуйста, с уважением ко времени, которое мы тратим на ревью вашей работы.